stores/evga.py

```diff
@@ -166,7 +167,7 @@ def buy(self, delay=5, test=False):
         selenium_utils.wait_for_page(self.driver, "EVGA - Checkout - Billing Options")
 
         log.info("Ensure that we are paying with credit card")
-        sleep(1)  # Fix this.
+        sleep(3)
         WebDriverWait(self.driver, 10).until(
             EC.element_to_be_clickable((By.XPATH, './/input[@value="rdoCreditCard"]'))
         ).click()
```

Comment:
Initially this is what I was doing, we scrolled to the element before trying to click it, but it was then suggested that we should only use WebDriverWait so I removed it. I dont know if the sleep alone will solve the issue, but I at least know the previous PR did solve the issue for @pauldofish. I'm not sure I care how it's resolved as long as it gets resolved.

Reply:
My hesistation with relying on arbitary delays is that they will fail once the site slows down due to large increases in traffic and the bot will end up giving up before the page has even fully loaded.

Reply:
Right, but what I don't get is why the WebDriverWait isn't actually doing what it says it should. The benchmarked page load times for me, were about 1.4 seconds. This caused line 171 to either fail or get skipped. When I added the move_to_element it was solved for me, but not for the reportee, that required also bumping the sleep limit. So we either need to just eat the fact that we need the sleep there, or figure out why WebDriverWait isn't actually waiting.

Reply:
One observation that I found was a little strange was that the exception handler was giving the message "Element is not clickable at point (508,920)". If I go into chrome devtools and bring up the coordinates of the radio-box we're trying to click on, I see that the radiobox is actually at (503,1081). Perhaps the coordinates of the radiobox are being slightly bumped around as the page is loading?

Reply:
@pauldofish yep, and that was one of the reasons for using move_to_element. Without comparing the browser dimensions it wouldn't know where it is, but if we just clamp to the element and navigate to it, it can shift around all it wants since we will pin to its position prior to sending the click event. Did you confirm that the latest PR resolved the issue or does it still exist?

Reply:
@worst-mod-confirmed the issue no longer comes up with the latest commit 👍